lib/matplotlib/animation.py

```diff
@@ -1166,13 +1166,13 @@ def _blit_draw(self, artists):
             view, bg = self._blit_cache.get(ax, (object(), None))
             if cur_view != view:
                 self._blit_cache[ax] = (
-                    cur_view, ax.figure.canvas.copy_from_bbox(ax.bbox))
+                    cur_view, ax.figure.canvas.copy_from_bbox(ax.figure.bbox))
         # Make a separate pass to draw foreground.
         for a in artists:
             a.axes.draw_artist(a)
         # After rendering all the needed artists, blit each axes individually.
         for ax in updated_ax:
-            ax.figure.canvas.blit(ax.bbox)
+            ax.figure.canvas.blit(ax.figure.bbox)
```

Comment:
This well effectively blit the whole figure number-of-axess times.

Reply:
Yes, I have been test it and I think with one call is enough. Maybe the solution is remove that final for and just get directly the figure.bbox and call blit just one time.